recipes/dbus/1.x.x/conanfile.py

```diff
@@ -134,8 +130,10 @@ def generate(self):
             tc.project_options["message_bus"] = bool(self.options.message_bus)
         if self.options.get_safe("dbus_user"):
             tc.project_options["dbus_user"] = str(self.options.dbus_user)
-        tc.project_options["system_pid_file"] = str(self.options.get_safe("system_pid_file", ""))
-        tc.project_options["system_socket"] = str(self.options.get_safe("system_socket", ""))
+        if self.options.system_pid_file:
```

Comment:
@uilianries I think maybe these ifs should all use `.get_safe()` to handle the case that the key is not present in options at all, or can that never happen?

Reply:
That can never happen, as it's not being deleted in the config_options/configure methods :)